gapic-generator-java/DEVELOPMENT.md

```diff
@@ -155,6 +155,32 @@ To generate a production GAPIC API:
+      ```
+   4. Run the above command.
+      
+      It shows nothing because it's waiting for a debugger to attach.
+
+3. In the local generator, start the Remote JVM Debug configuration and you can debug the generator in Intellij.
```

Comment:
Can you add an instruction to set a breakpoint at the main function of gapic-generator-java?

Reply:
done